### PR TITLE
Wigya/fix default doc page

### PR DESCRIPTION
**Pull request by Wigya**
### Description
This PR changes the default page to "core features" instead of the obscure index page

### Related Issue

#12 

### Changes Made

Delete core-features.md, move the content to index.md

### Video

https://github.com/user-attachments/assets/05595975-a2b0-4ad7-b939-e75e61da000c

### Checklist

- [x] I have tagged the issue in this PR.
- [x] I have attached necessary screenshots.
- [x] I have provided a short description of the PR.
- [x] I ran `yarn build` and build is successful
- [x] My code follows the style guidelines of this project.
- [x] I have added necessary documentation (if applicable)

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ const config = {
         docs: {
           routeBasePath: "/",
           sidebarPath: require.resolve("./sidebars.js"),
-          editUrl: "https://github.com/trypear/pearai-app/tree/main/docs",
+          editUrl: "https://github.com/trypear/pearai-documentation/blob/main/",
```
Comment (ICEPrey) [inline docusaurus.config.js:49]: I noticed you also changed the url here so will you close the other pr?

**Review by ICEPrey — APPROVED** [review]
Nice find

File: docusaurus.config.js (hunk 1 above)
Comment (ICEPrey) [reply docusaurus.config.js:49]: https://github.com/trypear/pearai-documentation/pull/11 this specific pr

Comment (Wigya) [reply docusaurus.config.js:49]: Yeah, it's closed now

**Review by Fryingpannn — APPROVED** [review]

**Pull request merged**